src/juxt/site/alpha/handler.clj

```diff
@@ -77,7 +77,7 @@
     ;; Protects resources from PUTs that are too large. If you need to
     ;; exceed this limitation, explicitly declare ::spin/max-content-length in
     ;; your resource.
-    (when-let [max-content-length (get resource ::http/max-content-length (Math/pow 2 16))]
+    (when-let [max-content-length (get resource ::http/max-content-length (Math/pow 2 80))]
```

Comment:
From 64KB to billions terabytes

Reply:
ah yes, this pr is supposed to be a draft :D don't think juxt github account supports draft prs though